### ISSUE TITLE
Keep track of previous executions like Allure reporter does

### ISSUE THREAD

**Issue by jprealini** [LironEr/cypress-mochawesome-reporter#163]
It would be great if this plugin allowed to keep track of every execution and show history of every test like Allure reporter does.

I remember having found some time ago (probably a couple of years) an article somewhere that described a way to customize mochawesome reporter to achieve that, and was able to configure it at that point in time. 

Now I would need to have that funcionality, but as usual, I didn't save the code or the link to the article.

**Comment by LironEr** [comment]
Yeah, it would be nice if this reporter could do that, if you have a way to do that please share (:

This plugin just connects Cypress & Mochawesome, so we might need to add some stuff to Mochawesome to do that.
